models/step.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    id: Joi
```

Comment:
is this field required ? `name` & `buildId` should be enough to identify it.

Reply:
This is what it's going to put into postgres. Every model has an ID field. https://github.com/screwdriver-cd/data-schema/blob/master/models/banner.js#L7